x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/data_tier_allocation_field/data_tier_allocation_field.tsx

```diff
@@ -53,85 +54,100 @@ export const DataTierAllocationField: FunctionComponent<Props> = ({ phase, descr
   const [formData] = useFormData({ watch: dataTierAllocationTypePath });
   const allocationType: DataTierAllocationType = get(formData, dataTierAllocationTypePath);
 
-  return (
-    <NodesDataProvider>
-      {({ nodesByRoles, nodesByAttributes, isUsingDeprecatedDataRoleConfig }) => {
-        const hasDataNodeRoles = Object.keys(nodesByRoles).some((nodeRole) =>
-          // match any of the "data_" roles, including data_content.
-          nodeRole.trim().startsWith('data_')
-        );
-        const hasNodeAttrs = Boolean(Object.keys(nodesByAttributes ?? {}).length);
-        const isCloudEnabled = cloud?.isCloudEnabled ?? false;
-
-        const renderNotice = () => {
-          switch (allocationType) {
-            case 'node_roles':
-              if (isCloudEnabled && phase === 'cold') {
-                const isUsingNodeRolesAllocation =
-                  !isUsingDeprecatedDataRoleConfig && hasDataNodeRoles;
-                const hasNoNodesWithNodeRole = !nodesByRoles.data_cold?.length;
-
-                if (isUsingNodeRolesAllocation && hasNoNodesWithNodeRole) {
-                  // Tell cloud users they can deploy nodes on cloud.
-                  return (
-                    <>
-                      <EuiSpacer size="s" />
-                      <CloudDataTierCallout />
-                    </>
-                  );
-                }
-              }
-
-              const allocationNodeRole = getAvailableNodeRoleForPhase(phase, nodesByRoles);
-              if (
-                allocationNodeRole === 'none' ||
-                !isNodeRoleFirstPreference(phase, allocationNodeRole)
-              ) {
-                return (
-                  <>
-                    <EuiSpacer size="s" />
-                    <DefaultAllocationNotice phase={phase} targetNodeRole={allocationNodeRole} />
-                  </>
-                );
-              }
-              break;
-            case 'node_attrs':
-              if (!hasNodeAttrs) {
-                return (
-                  <>
-                    <EuiSpacer size="s" />
-                    <NoNodeAttributesWarning phase={phase} />
-                  </>
-                );
-              }
-              break;
-            default:
-              return null;
+  const { data, resendRequest, error, isLoading } = useLoadNodes();
```

Comment:
Delete `error` here and on the line below it add:

```
const error = new Error('test')
```

to see the erroring state.